xmlWorksheet.go

```diff
@@ -231,11 +231,18 @@ type xlsxRow struct {
 // currently I have not checked it for completeness - it does as much
 // as I need.
 type xlsxC struct {
-	R string `xml:"r,attr"`           // Cell ID, e.g. A1
-	S int    `xml:"s,attr"`           // Style reference.
-	T string `xml:"t,attr,omitempty"` // Type.
-	V string `xml:"v"`                // Value
-	F *xlsxF `xml:"f,omitempty"`      // Formula
+	R  string  `xml:"r,attr"`           // Cell ID, e.g. A1
+	S  int     `xml:"s,attr"`           // Style reference.
+	T  string  `xml:"t,attr,omitempty"` // Type.
+	V  string  `xml:"v"`                // Value
+	F  *xlsxF  `xml:"f,omitempty"`      // Formula
+	IS *xlsxIS `xml:"is"`               // Inline string
```

Comment:
not sure when this struct is ever encoded, but should IS also have `,omitempty` set if it's not commonly used?